pkgs/top-level/all-packages.nix

```diff
@@ -26713,6 +26713,11 @@ with pkgs;
     tlsSupport = true;
     msVarsTemplate = stdenv.isx86_64 || stdenv.isAarch64;
   };
+  OVMF-cloud-hypervisor = callPackage ../applications/virtualization/OVMF {
```

Comment:
I'm really not sure what the best way to appease the by-name checker. I guess you could create `by-name/ov/OVMF-cloud-hypervisor.package.nix` that just does the callPackage. 🤷